internal/backend/crypto/age/identities.go

```diff
@@ -159,7 +159,7 @@ func (a *Age) addIdentity(ctx context.Context) ([]age.Identity, error) {
 }
 
 func (a *Age) saveIdentities(ctx context.Context, ids []string, newFile bool) error {
-	if !ctxutil.HasPasswordCallback(ctx) {
+	if !ctxutil.HasPasswordCallback(ctx) && ctxutil.IsInteractive(ctx) {
```

Comment:
So, if:
- we are NOT interactive
- and we do NOT have a password callback function

Then the `encryptFile` below would call the non-existent password callback:
https://github.com/gopasspw/gopass/blob/f6237d4003955fa808285eab9c91fc14d9d20fef/internal/backend/crypto/age/encrypt.go#L70-L74

And that would fail with 
https://github.com/gopasspw/gopass/blob/f6237d4003955fa808285eab9c91fc14d9d20fef/pkg/ctxutil/ctxutil.go#L431

This is not obvious at first, maybe a comment would be nice.